core/startos/src/service/service_effect_handler.rs

```diff
@@ -148,6 +151,7 @@ pub fn service_effect_handler() -> ParentHandler {
                 .no_display()
                 .with_remote_cli::<ContainerCliContext>(),
         )
+        .subcommand("setSystemSmtp", from_fn_async(set_system_smtp).no_cli())
```

Comment:
this should be an os rpc, not a service rpc

Reply:
That makes sense. This os rpc call should be in `core/startos/src/system.rs`, correct? 

Similarly, should `getSystemSmtp` also be an os rpc then?

Reply:
getSystemSmtp is a service rpc, so it is in the right place